orange/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM openjdk:17-alpine
```

Comment:
Are we sure that hmf-tools run on 17? I think we normally extend from `eclipse-temurin:11-jre`